Makefile

```diff
@@ -1,27 +1,28 @@
-egg-info:
-	pip install -q -e .
+app:
+	gunicorn -w 5 -b 0.0.0.0:8000 --log-level info --log-file /var/log/gunicorn.log aleph.manage:app
```

Comment:
The previous command was different in the `prod` file since we need support for hot-reload and other minor features when developing.